mllib/src/main/scala/org/apache/spark/mllib/fpm/PrefixSpan.scala

```diff
@@ -217,6 +217,7 @@ object PrefixSpan extends Logging {
       data: RDD[Array[Array[Item]]],
       minCount: Long): Array[Item] = {
 
+    data.persist()
```

Comment:
Just an user input, therefore persist not required. Its only caller
https://github.com/apache/spark/blob/80fbc382a60973db21367a922a0fb797e5ab382d/mllib/src/main/scala/org/apache/spark/mllib/fpm/PrefixSpan.scala#L146